CONTRIBUTING.md

```diff
@@ -0,0 +1,75 @@
+# Contribute to MiniPlaner
+
+
+We are happy about contributions to MiniPlaner in any form, be it new 
+functionality, documentation, bug reports, feature requests, or anything else.
+
+If you would like to contribute to MiniPlaner, a good start is looking for 
+[issues](https://github.com/yannikschaelte/MiniPlaner/issues), in particular 
+ones marked `good first issue` or `help wanted`. For other ideas or questions, 
+just post an issues.
+
+
+## Branches / releases
+
+MiniPlaner roughly follows the
+[GitFlow](https://nvie.com/posts/a-successful-git-branching-model/). All new
+contributions are merged into `develop`. These changes are regularly merged
+into `master` as new releases. For release versioning we are trying to follow
+[semantic versioning](https://semver.org/).
+
+We try to keep a clean git history. Therefore, feature pull requests are
+squash-merged to `develop`. Merging of release branches to master is done via
```

Comment:
About always squash-merging PRs:
Working on #33, there are at least two commits, which I would like to preserve in the commit history to keep it clean. I am not sure, if it is good to generalize this.

Reply:
sure, there may be exceptions where keeping the full history makes sense. but usually, it should be one aggregated commit per feature/pr to enhance readability and trackability of errors later.